.eslintrc.json

```diff
@@ -1,22 +0,0 @@
-{
-  "ecmaFeatures": {
-    "modules": true
-  },
-  "env": {
-    "es6": true,
-    "browser": true
-  },
-  "parserOptions": {
-    "ecmaVersion": 6,
-    "sourceType": "module"
-  },
-  "rules": {
-    "semi": 2,
-    "eqeqeq": 2,
-    "no-undef": 2,
-    "no-unused-vars": 1,
-    "no-use-before-define": 0,
-    "max-depth": [2, 3],
-    "no-loop-func": 2
```

Comment:
Maybe we should keep these?

Reply:
Already in Airbnb style. for max-depth, it will be very hard to have this one.

Reply:
Done for `no-use-before-define`